app/screens/Bible/VerseView.js

```diff
@@ -2,52 +2,38 @@ import React, { Component } from 'react';
 import {
   Text,
 } from 'react-native';
-import {connect} from 'react-redux'
-import {getResultText}  from  '../../utils/UtilFunctions'
-  
+import { connect } from 'react-redux'
+import { getResultText } from '../../utils/UtilFunctions'
+
 class VerseView extends Component {
-  constructor(props){
+  constructor(props) {
     super(props)
-      this.itemHeights =[]
-    // this.Animation = new Animated.Value(0);
-    // this.styles = styles(this.props.colorFile, this.props.sizeFile);    
+    this.itemHeights = []
 
   }
   onPress() {
     this.props.getSelection(
-        this.props.index, 
-        this.props.chapterNumber,
-        this.props.verseData.number,
-        this.props.verseData.text
+      this.props.index,
+      this.props.chapterNumber,
+      this.props.verseData.number,
+      this.props.verseData.text
     );
   }
-  // matchedVerse = () =>{
-  //   this.Animation.setValue(0);
-  //   Animated.timing(
-  //       this.Animation,
-  //       {
-  //           toValue: 1,
-  //           duration: 2000
-  //       }
-  //   ).start();
-  // }
-  // componentDidMount(){
-  //   this.matchedVerse()
-  // }
+
   has(selectedReferences, obj) {
-    for(var i = 0; i < selectedReferences.length; i++) {
+    for (var i = 0; i < selectedReferences.length; i++) {
       if (selectedReferences[i] == obj) {
         return true;
       }
     }
     return false;
   }
-  isHighlight(){
-      for(var i = 0 ; i<=this.props.HightlightedVerse.length; i++ ){
-        if(this.props.HightlightedVerse[i] == this.props.verseData.number){
-          return true
-          }
+  isHighlight() {
+    for (var i = 0; i <= this.props.HightlightedVerse.length; i++) {
+      if (this.props.HightlightedVerse[i] == this.props.verseData.number) {
+        return true
       }
+    }
     return false
   }
   // isNoted(){
```

Comment:
remove the commented lines if not needed

Reply:
this may need in next release for the making note icon touchable